docs/getting-started/how-prisma-schemas-are-converted-into-entities.md

```diff
@@ -186,7 +186,7 @@ Here is a table summarizing the Prisma to Amplication data type conversions:
 | DateTime (@default(now())) | CreatedAt | DateTime with now() default converts to CreatedAt |
 | DateTime (@updatedAt) | UpdatedAt | DateTime with updatedAt attribute converts to UpdatedAt |
 | DateTime | DateTime | Plain DateTime remains DateTime |
-| Float, Decimal | Decimal | Float and Decimal convert to Decimal |
+| Float, Decimal | Decimal Number | Float and Decimal convert to Decimal Number |
```

Comment:
I think it will be more clear to write in the description: "Amplication's Decimal Number data type". Same for the other descriptions.